internal/openstack/client.go

```diff
@@ -223,8 +223,20 @@ func getProvider(
 	authInfo.AuthURL = authURL
 	authInfo.Username = strings.TrimSpace(cfg.Section("Global").Key("username").String())
 	authInfo.Password = strings.TrimSpace(cfg.Section("Global").Key("password").String())
+
 	authInfo.DomainName = strings.TrimSpace(cfg.Section("Global").Key("domain-name").String())
-	authInfo.ProjectName = strings.TrimSpace(cfg.Section("Global").Key("tenant-name").String())
+	authInfo.DomainID = strings.TrimSpace(cfg.Section("Global").Key("domain-id").String())
+
+	legacyProjectName := strings.TrimSpace(cfg.Section("Global").Key("tenant-name").String())
```

Comment:
just a little nit but that would fit okay i guess :D
```suggestion
	tenantName := strings.TrimSpace(cfg.Section("Global").Key("tenant-name").String())
```

Reply:
@breuerfelix why? `tenant-name` is legacy

Reply:
oh that you mean. Seems more confusing to me